frontend/src/components/Layout/Footer.tsx

```diff
@@ -1,5 +1,6 @@
 import ButtonLink from '../Button/ButtonLink';
 import { useTranslation } from 'react-i18next';
+import { FaGithub } from 'react-icons/fa';
```

Comment:
Is the react-icons dependency useful for anything else?
If not maybe just use an svg version of the github logo

Reply:
Thank you, good idea, I will do that.